### PR TITLE
fix: payment token api definitions

### PR DESCRIPTION
**Pull request by danio1024**
## Summary
Added nullable where missing.

## Checklist

- [x] Writing style
- [x] API design standards

### CODE REVIEW

File: openapi/components/schemas/PaymentTokens/BankAccountToken.yaml [hunk 1]
```diff
@@ -39,10 +42,13 @@ properties:
     $ref: ../UpdatedTime.yaml
   usageTime:
     description: Date and time when the token is used.
+    nullable: true
     allOf:
       - $ref: ../ServerTimestamp.yaml
+
```
Comment (ashkarpetin) [inline openapi/components/schemas/PaymentTokens/BankAccountToken.yaml:48]: Why an empty line here?

File: openapi/components/schemas/PaymentTokens/DigitalWalletToken.yaml [hunk 2]
```diff
@@ -92,10 +96,13 @@ properties:
     $ref: ../UpdatedTime.yaml
   usageTime:
     description: Date and time when the token is used.
+    nullable: true
     allOf:
       - $ref: ../ServerTimestamp.yaml
+
```
Comment (ashkarpetin) [inline openapi/components/schemas/PaymentTokens/DigitalWalletToken.yaml:102]: Why an empty line here?

File: openapi/components/schemas/PaymentTokens/KhelocardCardToken.yaml [hunk 3]
```diff
@@ -62,10 +65,13 @@ properties:
     $ref: ../UpdatedTime.yaml
   usageTime:
     description: Date and time when the token is used.
+    nullable: true
     allOf:
       - $ref: ../ServerTimestamp.yaml
+
```
Comment (ashkarpetin) [inline openapi/components/schemas/PaymentTokens/KhelocardCardToken.yaml:71]: Why an empty line here?

File: openapi/components/schemas/PaymentTokens/AlternativePaymentToken.yaml [hunk 4]
```diff
@@ -45,10 +48,14 @@ properties:
     $ref: ../UpdatedTime.yaml
   usageTime:
     description: Date and time when the token is used.
+    type: string
+    nullable: true
     allOf:
       - $ref: ../ServerTimestamp.yaml
```
Comment (ashkarpetin) [inline openapi/components/schemas/PaymentTokens/AlternativePaymentToken.yaml:54]: I think you can replace allOf usage with inline definition

```
    format: date-time
    readOnly: true
```

**Review by ashkarpetin — APPROVED** [review]

**Review by jvc9109 — APPROVED** [review]

**Review by noon-dawg — APPROVED** [review]

**Pull request merged**